modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -11,32 +11,10 @@ Traditional clients are not supported.
 Canonical does not endorse or support {susemgr}.
 ====
 
-
-Supported features:
-
-* Bootstrapping
-* Synchronizing [systemitem]``.deb`` channels
-* Assigning [systemitem]``.deb`` channels to clients
-* GPG signing [systemitem]``.deb`` repositories
-* Information displayed in System details pages
-* Package install, update, and remove
-* Package install using [systemitem]``Package States``
-* Configuration and state channels
-
 Bootstrapping is supported for starting {ubuntu} clients and performing initial state runs such as setting repositories and performing profile updates.
```

Comment:
Why is this staying?

Reply:
Because it explains why you need to enable the root user. I've just put those two paras together into one so it makes more sense now.